### DISCUSSION TITLE
Ajouts à prévoir pour la version 1.2.0

### DISCUSSION THREAD

**Discussion by Nash115** [Nash115/frenchATC-for-MSFS2020#13] [Announcements]
- [ ] Ajouts prévus ici #33
- [ ] Corrections de bugs :
 -  - [ ]  Bug lors de vent variable (WIP)

Ajout de nouveaux aéroports :
- Paris Charles de gaulle (LFPG) (by @Nash115 )
- Toulouse Lasbordes (LFCL) (by [@philou3191](https://github.com/philou3191) )
- Niort Marais-poitevin (LFBN) (by [@zb2oby](https://github.com/zb2oby) )
- Alès-Deaux (LFMS) (by @Nash115 ) (asked by @Lebebert )

**Comment by Lebebert** [comment]
Ales-Deaux (LFMS)

**Reply by Nash115** [reply]
Ajouté dans le main, sera disponible dès la prochaine mise à jour

**Comment by Rbtsv2** [comment]
Salut Nash j'ai fais quelques mise à jour sur ton projet as tu discord ? Bien à toi rbtsv2

**Reply by Nash115** [reply]
Bonjour, oui avec plaisir ! mon discord c'est `nash115`
